recipe/time_util_const_dedup.patch

```diff
@@ -0,0 +1,54 @@
+diff --git src/google/protobuf/util/time_util.cc src/google/protobuf/util/time_util.cc
+index 031d019..9a4a53f 100644
+--- src/google/protobuf/util/time_util.cc
++++ src/google/protobuf/util/time_util.cc
+@@ -44,6 +44,15 @@ namespace util {
+ using google::protobuf::Timestamp;
+ using google::protobuf::Duration;
+ 
++// The min/max Timestamp/Duration values we support.
++//
++// For "0001-01-01T00:00:00Z".
++const int64 TimeUtil::kTimestampMinSeconds = -62135596800LL;
++// For "9999-12-31T23:59:59.999999999Z".
++const int64 TimeUtil::kTimestampMaxSeconds = 253402300799LL;
++const int64 TimeUtil::kDurationMinSeconds = -315576000000LL;
++const int64 TimeUtil::kDurationMaxSeconds = 315576000000LL;
```

Comment:
Assigning the values for these in the source file as explained in the other comment.